app/javascript/dashboard/routes/dashboard/conversation/ConversationInfo.vue

```diff
@@ -1,152 +1,103 @@
+    props.conversationAttributes.browser;
+  return `${browser} ${browserVersion}`;
+});
+const browserLanguage = computed(() =>
+  getLanguageName(props.conversationAttributes.browser_language)
+);
+const platformName = computed(() => {
+  const { browser } = props.conversationAttributes;
+  if (!browser) {
+    return '';
+  }
+  const { platform_name: platform, platform_version: platformVersion } =
+    browser;
+  return `${platform || ''} ${platformVersion || ''}`;
+});
+
+const createdAtIp = computed(() => props.contactAttributes.created_at_ip);
+
+const staticElements = computed(() => {
+  const conversationAttributes = [
     {
-      name: 'initiated_at',
-      label: 'CONTACT_PANEL.INITIATED_AT',
+      content: initiatedAt,
+      title: 'CONTACT_PANEL.INITIATED_AT',
     },
     {
-      name: 'referer',
-      label: 'CONTACT_PANEL.BROWSER',
+      content: browserLanguage,
+      title: 'CONTACT_PANEL.BROWSER_LANGUAGE',
     },
     {
-      name: 'browserName',
-      label: 'CONTACT_PANEL.BROWSER',
+      content: referer,
+      title: 'CONTACT_PANEL.INITIATED_FROM',
+      type: 'link',
     },
     {
-      name: 'platformName',
-      label: 'CONTACT_PANEL.OS',
+      content: browserName,
+      title: 'CONTACT_PANEL.BROWSER',
     },
     {
-      name: 'ipAddress',
-      label: 'CONTACT_PANEL.IP_ADDRESS',
-    },
-  ],
-  computed: {
-    referer() {
-      return this.conversationAttributes.referer;
-    },
-    initiatedAt() {
-      return this.conversationAttributes.initiated_at;
-    },
-    browserName() {
-      if (!this.conversationAttributes.browser) {
-        return '';
-      }
-      const {
-        browser_name: browserName = '',
-        browser_version: browserVersion = '',
-      } = this.conversationAttributes.browser;
-      return `${browserName} ${browserVersion}`;
+      content: platformName,
+      title: 'CONTACT_PANEL.OS',
     },
-    browserLanguage() {
-      return getLanguageName(this.conversationAttributes.browser_language);
-    },
-    platformName() {
-      if (!this.conversationAttributes.browser) {
-        return '';
-      }
-      const { platform_name: platformName, platform_version: platformVersion } =
-        this.conversationAttributes.browser;
-      return `${platformName || ''} ${platformVersion || ''}`;
-    },
-    ipAddress() {
-      const { created_at_ip: createdAtIp } = this.contactAttributes;
-      return createdAtIp;
-    },
-    customAttributeRowClass() {
-      const attributes = [
-        'initiatedAt',
-        'referer',
-        'browserName',
-        'platformName',
-        'ipAddress',
-      ];
-      const availableAttributes = attributes.filter(
-        attribute => !!this[attribute]
-      );
-      return availableAttributes.length % 2 === 0 ? 'even' : 'odd';
+    {
+      content: createdAtIp,
+      title: 'CONTACT_PANEL.IP_ADDRESS',
     },
-  },
-};
+  ];
+  return conversationAttributes.filter(attribute => !!attribute.content.value);
+});
```

Comment:
Can we remove the unnecessary computations inside the computed property like this?

```
const staticElements = computed(() => [
  {
    content: initiatedAt,
    title: 'CONTACT_PANEL.INITIATED_AT',
  },
  {
    content: browserLanguage,
    title: 'CONTACT_PANEL.BROWSER_LANGUAGE',
  },
  {
    content: referer,
    title: 'CONTACT_PANEL.INITIATED_FROM',
    type: 'link',
  },
  {
    content: browserName,
    title: 'CONTACT_PANEL.BROWSER',
  },
  {
    content: platformName,
    title: 'CONTACT_PANEL.OS',
  },
  {
    content: createdAtIp,
    title: 'CONTACT_PANEL.IP_ADDRESS',
  },
].filter(attribute => !!attribute.content.value));
```